tap_hubspot/hubspot.py

```diff
@@ -677,14 +673,15 @@ def merge_guids():
                 self.test_endpoint(path)
             except:
                 continue
-            yield from self.get_records(
+            for record in self.get_records(
```

Comment:
get_records return `record and replication_value`
```suggestion
            for record,replication_value in self.get_records(
```